tests/alignmentcommand.js

```diff
@@ -0,0 +1,162 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import AlignmentCommand from '../src/alignmentcommand';
+
+import { getData as getModelData, setData as setModelData } from '@ckeditor/ckeditor5-engine/src/dev-utils/model';
+
+import Command from '@ckeditor/ckeditor5-core/src/command';
+import ModelTestEditor from '@ckeditor/ckeditor5-core/tests/_utils/modeltesteditor';
+
+describe( 'AlignmentCommand', () => {
+	let editor, doc, command, defaultAlignmentCommand;
+
+	beforeEach( () => {
+		return ModelTestEditor.create()
+			.then( newEditor => {
+				doc = newEditor.document;
+				command = new AlignmentCommand( newEditor, 'center' );
+				defaultAlignmentCommand = new AlignmentCommand( newEditor, 'left' );
+				editor = newEditor;
+
+				editor.commands.add( 'alignCenter', command );
+				editor.commands.add( 'alignLeft', defaultAlignmentCommand );
+
+				doc.schema.registerItem( 'paragraph', '$block' );
+
+				doc.schema.allow( { name: '$block', inside: '$root', attributes: 'alignment' } );
+			} );
+	} );
+
+	afterEach( () => {
+		editor.destroy();
+	} );
+
+	it( 'is a command', () => {
+		expect( AlignmentCommand.prototype ).to.be.instanceOf( Command );
+		expect( command ).to.be.instanceOf( Command );
+	} );
+
+	describe( 'value', () => {
+		it( 'is true when selection is in block with commend type alignment', () => {
+			setModelData( doc, '<paragraph alignment="center">x[]x</paragraph>' );
+
+			expect( command ).to.have.property( 'value', true );
+		} );
+
+		it( 'is false when selection is not block that has different alignment', () => {
+			setModelData( doc, '<paragraph alignment="justify">x[]x</paragraph>' );
+
+			expect( command ).to.have.property( 'value', false );
+		} );
+
+		it( 'is true when selection is in block with default alignment', () => {
+			setModelData( doc, '<paragraph>x[]x</paragraph>' );
+
+			expect( defaultAlignmentCommand ).to.have.property( 'value', true );
+		} );
+
+		it( 'is false when selection is not block that has different alignment', () => {
```

Comment:
"is false when selection is inside block that has different alignment (default style)" WDYT?